sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2210,13 +2210,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
-  val LEGACY_ADD_DIRECTORY_USING_RECURSIVE =
-    buildConf("spark.sql.legacy.addDirectory.recursive.enabled")
+  val LEGACY_ADD_SINGLE_FILE_IN_ADD_FILE =
+    buildConf("spark.sql.legacy.addSingleFileInAddFile")
       .internal()
-      .doc("When true, users can add directory by passing path of a directory to ADD FILE " +
-        "command of SQL. If false, then only a single file can be added.")
+      .doc("When true, only a single file can be added using ADD FILE. If false, then users " +
+        "can add directory by passing directory path to ADD FILE.")
       .booleanConf
-      .createWithDefault(true)
+      .createWithDefault(false)
```

Comment:
? Why do you try to change the default value in `Rename ...` PR title?

Reply:
Yea, I think this comes from the comment: https://github.com/apache/spark/pull/26863#discussion_r378715586

Reply:
Thanks, @maropu .